src/main/java/com/nike/cerberus/auth/connector/onelogin/OneLoginAuthConnector.java

```diff
@@ -63,6 +63,12 @@ public AuthResponse authenticate(String username, String password) {
         return authResponse;
     }
 
+    @Override
+    public AuthResponse triggerChallenge(String stateToken, String deviceId) {
+
+        throw new UnsupportedOperationException("Method call is not allowed.");
```

Comment:
This will be an uncaught error that will show up to a user as a generic internal server error, when this seems like a bad request to me.

Perhaps an API error with a usable error message here would be better.

Reply:
- [ ] Make this an API error with good error message.

Reply:
I changed the error from Unsupported Operation to an API error. I was unable to make it a 501 because the ProjectAPIErrorsTestBase that we use did not include that error, so I made it a 405 Method Not Allowed.